doublyLinkedList.js

```diff
@@ -103,6 +103,87 @@ class DoublyLinkedList {
 
     return null;
   }
+  
+  sort(isAscending = true) {
```

Comment:
Could you please explain, what the type of sortiring you chose?

Reply:
It's a selection sort. Subfunction find max or min element, then finded node move to the end of list. On  the next steps it doesn't check moved elements

Reply:
What is the complexity of this sort type?
What kind of sort types you know?
Maybe we have better options to sort this data structure?

Reply:
The complexity of this sort type is O(n^2).
I know bubble sort, selection sort, quick sort and insertion sort. I think that I can't use quick sort for list, because it's recursive sort.
Maybe yes, but I don't know.

Reply:
Merge sort is better?